srcpkgs/docker/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'docker'
 pkgname=docker
 version=18.09.3
-revision=1
+revision=2
 wrksrc="${pkgname}-ce-${version}"
 hostmakedepends="git go pkg-config curl cmake"
-makedepends="libbtrfs-devel sqlite-devel device-mapper-devel libseccomp-devel libapparmor-devel libltdl-devel"
+makedepends="libbtrfs-devel sqlite-devel device-mapper-devel libseccomp-devel
+ libapparmor-devel libltdl-devel"
+depends="containerd runc"
```

Comment:
There's a second depends variable on line 18 -- either that should be updated or its packages should be moved into this one.

Reply:
relatedly, why does it require git at runtime??

Reply:
`docker build <path|url>` takes either a path to a context or a URL to a git repository. Wondered the same a while ago -- I don't know how many people use that feature, but it's there.